pyccel/codegen/codegen.py

```diff
@@ -135,12 +143,21 @@ def language(self):
         return self._language
 
     def set_printer(self, **settings):
-        """ Set the current codeprinter instance"""
+        """
+        Set the current codeprinter instance
+        
+        Getting the language that will be used (default language used is fortran),
+        Then instantiating the codePrinter with the corresponding language.
+        Parameters
```

Comment:
```suggestion
        Then instantiating the codePrinter with the corresponding language.

        Parameters
```
Without the space numpydoc cannot detect the start of the Parameters section

Reply:
[f3f1e08](https://github.com/pyccel/pyccel-cuda/pull/32/commits/f3f1e0800f1c23411200943ccf362feee1cb084c)